app/views/error.html

```diff
@@ -0,0 +1,7 @@
+<div class="container" id="errorPage">
+  <div class="well">
+    <h2>Oops! We Couldn't find what you were looking for.</h2>
```

Comment:
This error seems too specific. It may not be that they were looking for something at all...think more generic.